apis/virtualKubelet/v1alpha1/namespacemap_types.go

```diff
@@ -20,24 +20,28 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type RemoteNamespaceStatus struct {
+	// RemoteName is the name chosen by the user at creation time (when he puts labels on his local namespace)
+	RemoteName string `json:"remoteName,omitempty"`
+	// RemoteStatus is the remote Namespace's actual status (READY,FAILED)
+	RemoteStatus string `json:"remoteStatus,omitempty"`
```

Comment:
Usually, the term phase is used in this context. Therefore I propose something like `Phase`. Besides, I propose to use a custom type for `Phase` instead of string. something like `type mappingPhase string` and create the constants you need to describe all the possible phases.

Reply:
Thank you for the advise